internal/apis/controlplane/kubeadm/v1alpha3/conversion.go

```diff
@@ -154,3 +155,11 @@ func Convert_v1alpha3_KubeadmControlPlaneSpec_To_v1beta1_KubeadmControlPlaneSpec
 	out.MachineTemplate.NodeDrainTimeout = in.NodeDrainTimeout
 	return autoConvert_v1alpha3_KubeadmControlPlaneSpec_To_v1beta1_KubeadmControlPlaneSpec(in, out, s)
 }
+
+func Convert_v1beta1_KubeadmConfigSpec_To_v1alpha3_KubeadmConfigSpec(in *bootstrapv1.KubeadmConfigSpec, out *bootstrapv1alpha3.KubeadmConfigSpec, s apiconversion.Scope) error {
```

Comment:
This is a workaround for now. Brought up the issue here: https://github.com/kubernetes/code-generator/issues/94